### ISSUE TITLE
Support transparency

### ISSUE THREAD

**Issue by hsayed21** [Toinane/colorpicker#136]
Support transparency

**Comment by Toinane** [comment]
Hey @hsayed21 What do you mean about transparency? Do you want the window to become transparent? If it's the case, older version of Colorpicker had this feature but was really buggy, I've removed it because of that. I can't do anything because it's an issue from Electron.

**Comment by hsayed21** [comment]
No, I mean the color transparency (Alpha value for color) with RGB to be RGBA
